assets/wizards/advertising/views/services/index.js

```diff
@@ -68,6 +68,22 @@ const Services = ( { services, toggleService } ) => {
 				toggleOnChange={ value => toggleService( 'google_ad_manager', value ) }
 				titleLink={ google_ad_manager ? '#/google_ad_manager' : null }
 				href={ google_ad_manager && '#/google_ad_manager' }
+				notification={
+					google_ad_manager.error
+						? [ google_ad_manager.error ]
+						: google_ad_manager.created_targeting_keys?.length > 0 && [
+								__( 'Created custom targeting keys:' ) + '\u00A0',
+								google_ad_manager.created_targeting_keys.join( ', ' ) + '. \u00A0',
+								// eslint-disable-next-line react/jsx-indent
```

Comment:
I've disabled lint here because the linter is requiring one less tab, which does not make sense to me.